packages/native/src/components/Form/SelectableList.tsx

```diff
@@ -40,13 +56,22 @@ function Element<V>({
         alignItems={"center"}
       >
         {Icon && (
-          <Flex mr={6}>
+          <Flex mr={6} flexShrink={0}>
             <Icon size={24} color={disabled ? "neutral.c50" : "neutral.c100"} />
           </Flex>
         )}
         <Text variant="large" color={disabled ? "neutral.c50" : "neutral.c100"}>
           {children || value}
         </Text>
+        {RenderRight && (
+          <Flex pl={6} ml={"auto"} flexShrink={0}>
```

Comment:
Since we are lucky to be inside a flexbox here I think that it would be better to drop this `ml="auto"` here and just put a `flex: 1` on the middle element to make it responsive instead of mixing up 2 concepts (margin auto + flex)

Reply:
sold

Reply:
I just added baseStyled on text as Flex wasn't supporting flex props

Reply:
Noice👌